example_python_in_jupyter.ipynb

```diff
@@ -83,7 +83,8 @@
    "source": [
```

Comment:
<p>not sure what's going on here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-sebkopf/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>